ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -21,15 +21,36 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
 
   // =attributes
 
-  queryParams = ['search', 'page', 'pageSize'];
+  queryParams = ['search', { scopes: { type: 'array' } }, 'page', 'pageSize'];
 
+  allTargets;
   @tracked search;
+  @tracked scopeSearchTerm;
+  @tracked scopes = [];
+  @tracked selectedScopes = [];
   @tracked page = 1;
   @tracked pageSize = 10;
-  @tracked searchItems = ['Project 1', 'Project 2', 'Project 3'];
+
+  constructor() {
+    super(...arguments);
+    this.getAllTargets();
+  }
 
   // =methods
 
+  async getAllTargets() {
+    // Query all targets for defining filtering values
+    const options = { pushToStore: false };
+    let allTargets = await this.store.query('target', { query: null }, options);
```

Comment:
Why are we passing null? What happens when we don't pass anything to query?

Reply:
Updated to be just `{}`. Much nicer looking.